database/postStore.go

```diff
@@ -17,10 +17,20 @@ type Post struct {
 	Shared    []primitive.DBPointer `bson:"shared,omitempty" json:"shared,omitempty"`
 }
 
+type PostStore struct {
+	db *mongo.Database
+}
+
+func NewPostStore(db *mongo.Database) *PostStore {
+	return &PostStore{
+		db: db,
+	}
+}
+
 const POST_COLL = "posts"
```

Comment:
I really don't like the name of this constant. It should be verbose because it's global, but capitalization doesn't get us there.